src/libs/blueprint/blueprint_mesh.cpp

```diff
@@ -94,8 +133,504 @@ bool
 verify(const Node &n,
        Node &info)
 {
-    return false;
+    bool res = true;
+
+    // note "errors" is a list, this allows us to log multiple errors.
+    // Given that not conforming result is likely to trigger an error 
+    // state in client code it seems like we should give as much info as
+    // possible about what is wrong with the mesh, so we don't early
+    // return when an error is found.
+    
+    // required: "coordsets", with at least one child
+    //  each child must conform to "mesh::coordset"
+    if(!n.has_child("coordsets"))
+    {
+        info["errors"].append().set("missing child node \"coordsets\"");
+        res = false;
+    }
+    else if(n["coordsets"].number_of_children() == 0)
+    {
+        info["errors"].append().set("\"coordsets\" has no children");
+        res = false;
+    }
+    else
+    {
+        NodeConstIterator itr = n["coordsets"].children();
+    
+        while(itr.has_next())
+        {
+            const Node &chld = itr.next();
+            const std::string chld_name = itr.path();
+
+            if(!coordset::verify(chld,info["coordsets/" + chld_name]))
+            {
+                info["errors"].append().set(chld_name 
+                                + " is not a valid blueprint::mesh::coordset");
+                res = false;
+            }
+        }
+    }
+    
+    // required: "topologies",  with at least one child
+    // each child must conform to "mesh::topology"
```

Comment:
The functionality below for verifying the `topologies` path is nearly identical to that used above to verify the `coordsets` path.  The commonalities between these two segments should be factored out if possible.